Makefile

```diff
@@ -41,250 +25,224 @@ COMMON_ARGS += --build-arg=TOOLS=$(TOOLS)
 COMMON_ARGS += --build-arg=SHA=$(SHA)
 COMMON_ARGS += --build-arg=TAG=$(TAG)
 COMMON_ARGS += --build-arg=GO_VERSION=$(GO_VERSION)
-COMMON_ARGS += .
+COMMON_ARGS += --build-arg=IMAGES=$(ARTIFACTS)/images
 
-TESTPKGS ?= ./...
+all: help
+
+# Help Menu
+
+define HELP_MENU_HEADER
+# Getting Started
+
+To build this project, you must have the following installed:
+
+- git
+- make
+- docker (19.03 or higher)
+- buildx (https://github.com/docker/buildx)
+
+## Creating a Builder Instance
 
-all: builddeps rootfs initramfs kernel osctl-linux osctl-darwin installer container
+The build process makes use of features not currently supported by the default
+builder instance (docker driver). To create a compatible builder instance, run:
 
-.PHONY: builddeps
-builddeps: gitmeta
+```
+docker buildx create --driver docker-container --name local --use
```

Comment:
should add ` --buildkitd-flags '--allow-insecure-entitlement security.insecure'` here to allow unit-tests

Reply:
Thanks. 👍